invenio_collections/percolator.py

```diff
@@ -29,12 +31,16 @@ def new_collection_percolator(target):
     query = IQ(target.dbquery)
     for name in current_search.mappings.keys():
         if target.name and target.dbquery:
-            current_search.client.index(
-                index=name,
-                doc_type='.percolator',
-                id='collection-{}'.format(target.name),
-                body={'query': query.to_dict()}
-            )
+            try:
+                current_search.client.index(
+                    index=name,
+                    doc_type='.percolator',
+                    id='collection-{}'.format(target.name),
+                    body={'query': query.to_dict()}
+                )
+            except TransportError:
```

Comment:
@jmartinm wouldn't be better to introduce config to exclude creation of percolators in certain indexes?

```python
re_exclude_mappings = re.compile('^incompatible.*')
...
if re_exclude_mappings.match(name):
    continue
...
```

If you catch the exception without logging admins will not be aware that something is wrong.